### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by joshua-s**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -100,31 +100,30 @@ If you have any suggestions then please open an issue!
...
-derives a MasterPassKey from it which it then uses to derive the encryption key
-(which is used directly to encrypt your file). Every time a file is decrypted,
+Crypter derives a MasterPassKey from the MasterPass obtained at setup by using
+the PBKDF2 key derivation algorithm (see below for the specification). It
+then uses PBKDF2 to derive a number of encryption keys from the MasterPassKey 
+that can be used for the encryption of files. This method allows for the 
+generation of very secure encryption keys for data encryption. Moreover, by 
+publicly storing the credentials used to derive the MasterPassKey and the salts
+used to derive the encryption keys, you are able to produce the encryption keys 
+at will and without needing to store them securely. Your only task is to
```
Comment (HR) [inline readme.md]: "Your only task is..." seems a bit imperative and unwelcoming

File: readme.md [hunk 2]
```diff
@@ -184,177 +184,152 @@ someFile.crypto
...
-key on is **not the same** as the MasterPassKey derived with the reused
-MasterPass because the set of credentials generated with the original MasterPass
-is **different** (due to randomness). As a result your encryption key that is
-derived from the MasterPassKey is different and so incorrect which yields the
-error.
+This issue occurs because the MasterPassKey that was originally used to 
+derive the encryption key on is **not the same** as the MasterPassKey 
+derived with the reused MasterPass. Since Crypter uses randomness to 
+generate secure credentials, this second set of credentials will be quite 
+**different** from the original set.  As a result, your encryption key is
```
Comment (HR) [inline readme.md]: To be unambiguous, it should be "As a result, the derived encryption key..."

File: readme.md [hunk 3]
```diff
@@ -184,177 +184,152 @@ someFile.crypto
...
-file and the original encryption such as the encryption key and path of the
-decrypted file in a new screen. By default, the decrypted file has "Decrypted"
-with a space prepended to its file name with the original extension.
-
-### Why am I getting `Corrupted Crypter file or trying to decrypt on a different
+decrypt is being used with the same MasterPass that you set at setup and also
+that you have not reset it since that time. If this is not the case, please refer 
+to [Reusing the same MasterPass](#reusing-the-same-masterpass).
+
+To decrypt a CRYPTO file, you will need to launch Crypter and verify it.
```
Comment (HR) [inline readme.md]: *verify the MasterPass

**Comment by joshua-s** [comment]
@HR I resolved those issues 😃

**Pull request merged**